overlays/vault-pki-overlay.yaml

```diff
@@ -1,19 +1,34 @@
 applications:
   easyrsa: null
+  mysql-innodb-cluster:
+    channel: 8.0/stable
+    charm: mysql-innodb-cluster
+    constraints: cores=2 mem=8G root-disk=64G
+    num_units: 3
+    options:
+      enable-binlogs: true
+      innodb-buffer-pool-size: 256M
+      max-connections: 2000
+      wait-timeout: 3600
   vault:
+    channel: 1.7/stable
```

Comment:
Is there a reason to pin the charms to a specific charmhub channel?

i'm guessing using `latest/stable` doesn't work here?

Reply:
eh, `latest/stable` might work, but it's [3mo older](https://charmhub.io/vault) than `1.7/stable`. Similarly `mysql-innodb-cluster`'s latest stable is 4mo older than `8.0/stable`.

I went with the most recent stable release track for these o7k charms because we know these work with the current CK release.  It's a little tighter than `latest` and a little looser than a pinned `revision`. Balance in all things, yes?